chips/imxrt1050/src/lib.rs

```diff
@@ -0,0 +1,271 @@
+//! Peripheral implementations for the IMXRT1050 MCU.
+//!
+//! imxrt1050 chip: <https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/i-mx-rt1050-evaluation-kit:MIMXRT1050-EVK>
+
+#![crate_name = "imxrt1050"]
+#![crate_type = "rlib"]
+#![feature(llvm_asm, const_fn, in_band_lifetimes)]
```

Comment:
As part of the effort to move to stable Rust, we have removed all uses of the in_band_lifetimes feature in Tock. Can you remove it from this PR please?